internal/venice-test-common/src/main/java/com/linkedin/venice/testng/VeniceTestListener.java

```diff
@@ -11,14 +11,18 @@ public class VeniceTestListener extends TestListenerAdapter {
 
   @Override
   public void onTestStart(ITestResult result) {
+    super.onTestStart(result);
+    boolean interruptCleared = Thread.interrupted();
```

Comment:
Can we just mentioned here that  `Thread.interrupted() clears interrupt status?

Reply:
Sure, one sec...

Reply:
Thanks!